## 1> สร้าง function ชื่อ smartDict  ซึ่งทำหน้าที่:
- function รับ input เป็น list 2 ตัวที่มีขนาดเท่ากัน โดย list แรกเป็น list ของตัวเลข list ที่สองเป็น list ของ text 
- ถ้า list แรกเป็นเลขคู่ ให้ค่าใน dict ที่เป็น output มี key เป็นค่าจาก list ตัวที่ 1 และ value จาก list ตัวที่ 2
- ถ้า list แรกเป็นเลขคี่ ให้ค่าใน dict ที่เป็น output มี key เป็นค่าจาก list ตัวที่ 2 และ value จาก list ตัวที่ 1
- output ของ function คือ dict ที่สร้างขึ้น
เช่น

output1 = smartDict( [ 1 , 2 ,3 ] , [ 'apple' , 'banana' , 'orange'] , True )

จะได้ output1 เป็น { 1:'apple' , 'banana' : 2, 3 :'orange' }



In [240]:
def smartDict(listtext, listnumber, logicV):
    if logicV : #logicv == True
        retdict = {} #สร้างdictเปล่าขึ้นมา
        for index in range(len(listtext)):  
            retdict[listtext[index]]= listnumber[index] #dictทีละตัว
    else:
        retdict = {} 
        for index in range(len(listtext)):  
             retdict[listnumber[index]]= listtext[index] 
                    
            
    return retdict  #ส่งค่าoutput

In [241]:
smartDict( [ 1 , 2 ,3 ] , [ 'apple' , 'banana' , 'orange'] , True )

{1: 'apple', 2: 'banana', 3: 'orange'}

In [242]:
smartDict( [ 1 , 2 ,3 ] , [ 'apple' , 'banana' , 'orange'] , False)

{'apple': 1, 'banana': 2, 'orange': 3}

##  2> จากไฟล์ Wuhan_nCoV_GNome.txt จงเขียนโปรแกรมเพื่อตอบคำถามต่อไปนี้
- กดอมิโนใด (A, T, G, C) มีปริมาณมากทีุ่ดและมีเท่าไร
- จงหา sequence ที่มีขนาด 3 และ 4 ที่เกิดขึ้นซ้ำมากที่สุด


In [243]:
import pandas as pd

In [244]:
DNAseq = pd.read_csv('C:\\Users\\N-PC\\Documents\\python\\python_dataviz\\Wuhan_nCoV_GNome.txt',header=None)

In [245]:
DNAseq.head()

0
0  ATTAAAGGTTTATACCTTCCCAGGTAACAAACCAACCAACTTTCGA...
1  CGAACTTTAAAATCTGTGTGGCTGTCACTCGGCTGCATGCTTAGTG...
2  TAATTACTGTCGTTGACAGGACACGAGTAACTCGTCTATCTTCTGC...
3  TTGCAGCCGATCATCAGCACATCTAGGTTTCGTCCGGGTGTGACCG...
4  CCTGGTTTCAACGAGAAAACACACGTCCAACTCAGTTTGCCTGTTT...

In [246]:
DNAseq

0
0    ATTAAAGGTTTATACCTTCCCAGGTAACAAACCAACCAACTTTCGA...
1    CGAACTTTAAAATCTGTGTGGCTGTCACTCGGCTGCATGCTTAGTG...
2    TAATTACTGTCGTTGACAGGACACGAGTAACTCGTCTATCTTCTGC...
3    TTGCAGCCGATCATCAGCACATCTAGGTTTCGTCCGGGTGTGACCG...
4    CCTGGTTTCAACGAGAAAACACACGTCCAACTCAGTTTGCCTGTTT...
..                                                 ...
423  GTGCAGAATGAATTCTCGTAACTACATAGCACAAGTAGATGTAGTT...
424  TTAATCAGTGTGTAACATTAGGGAGGACTTGAAAGAGCCACCACAT...
425  GATCGAGTGTACAGTGAACAATGCTAGGGAGAGCTGCCTATATGGA...
426  TTTAGTAGTGCTATCCCCATGTGATTTTAATAGCTTCTTAGGAGAA...
427                                      AAAAAAAAAAAAA

[428 rows x 1 columns]

#### คำถามที่1 - กดอมิโนใด (A, T, G, C) มีปริมาณมากที่สุดและมีเท่าไร

In [247]:
DNAseq.shape

(428, 1)

In [248]:
DNAseq.iloc[1][0]

'CGAACTTTAAAATCTGTGTGGCTGTCACTCGGCTGCATGCTTAGTGCACTCACGCAGTATAATTAATAAC'

In [249]:
DNAlong = 0 
for row in range(DNAseq.shape[0]):
    DNAlong = DNAlong + len(DNAseq.iloc[row][0]) #DNAlong += en(DNAseq.iloc[row][0]
    
print(f'sequence is {DNAlong} charaters long')

sequence is 29903 charaters long


In [250]:
def countChar(seq,character):
    charcount = 0
    for char in seq:
         if char == character:
                charcount = charcount + 1
    return charcount

In [251]:
countChar('ATTAAAGGTTTATACCTTCCCAGGTAACAAACCAACCAACTTTCGA','A')

17

In [252]:
ANum = 0 
TNum = 0
GNum = 0 
CNum = 0 
for row in range(DNAseq.shape[0]):
    ANum += countChar(DNAseq.iloc[row][0],'A') 
    TNum += countChar(DNAseq.iloc[row][0],'T')
    GNum += countChar(DNAseq.iloc[row][0],'G')
    CNum += countChar(DNAseq.iloc[row][0],'C')
    
print(f'There are {ANum} As, {TNum} Ts, {GNum} Gs, {CNum} Cs')

There are 8954 As, 9594 Ts, 5863 Gs, 5492 Cs


กรดอะมิโนที่มีปริมาณมากที่สุดคือ T และมีค่ามากที่สุดคือ 9594

#### คำถามที่ 2 - จงหา sequence ที่มีขนาด 3  ที่เกิดขึ้นซ้ำมากที่สุด

In [253]:
def populate3charstring(seq):
    str_list = list()
    for i in range(len(seq)-4):
        str_list.append(seq[i:i+3])
    return str_list

In [254]:
populate3charstring('ATTAAAGGTTT')

['ATT', 'TTA', 'TAA', 'AAA', 'AAG', 'AGG', 'GGT']

In [255]:
list_all_3char = list()
for row in range(DNAseq.shape[0]):
    list_all_3char = list_all_3char + populate3charstring(DNAseq.iloc[row][0]) 

In [256]:
len(list_all_3char)

28191

In [257]:
unique_3char = list(set(list_all_3char))

In [258]:
len(unique_3char)

64

In [259]:
dict_3char = {}
for sub_str in unique_3char:
    num = 0
    for s in list_all_3char:
        if sub_str == s:
            num += 1
    dict_3char[sub_str] = num

In [260]:
dict_3char 

{'GCG': 86,
 'CAT': 460,
 'TAT': 582,
 'TTC': 488,
 'ACA': 773,
 'TAA': 666,
 'TTA': 820,
 'ATA': 443,
 'CGG': 73,
 'CGA': 89,
 'GCA': 355,
 'GGG': 126,
 'ACT': 641,
 'AGA': 567,
 'AAA': 865,
 'TCC': 193,
 'GGA': 266,
 'TTT': 955,
 'GGC': 207,
 'GCT': 488,
 'ACC': 354,
 'ATC': 326,
 'CAC': 436,
 'CCG': 70,
 'CTG': 466,
 'GTG': 515,
 'TGC': 519,
 'CAA': 661,
 'TCA': 521,
 'ACG': 151,
 'CTT': 700,
 'ATT': 727,
 'AAT': 721,
 'GAC': 319,
 'CTC': 269,
 'TGT': 809,
 'GAT': 416,
 'TTG': 773,
 'GAG': 284,
 'GGT': 430,
 'CCC': 108,
 'GTA': 438,
 'GTC': 248,
 'CCA': 324,
 'TAG': 401,
 'TCG': 107,
 'CTA': 524,
 'AAG': 549,
 'AAC': 577,
 'TGG': 526,
 'TGA': 589,
 'TCT': 507,
 'GTT': 662,
 'CGC': 90,
 'AGC': 288,
 'ATG': 690,
 'CGT': 161,
 'GCC': 178,
 'CCT': 329,
 'TAC': 570,
 'CAG': 417,
 'AGG': 313,
 'GAA': 507,
 'AGT': 478}

In [261]:
maxVal = -100
maxKey = None
for k in dict_3char.keys():
    if dict_3char[k] > maxVal:
        maxVal = dict_3char[k]
        maxKey = k

In [262]:
maxKey

'TTT'

In [369]:
print(f'max 3-char-sequence is {maxKey} with frequency equals {maxKey}')

max 3-char-sequence is TGTT with frequency equals TGTT


#### คำถามที่ 2 - จงหา sequence ที่มีขนาดะ 4 ที่เกิดขึ้นซ้ำมากที่สุด

In [264]:
def populate4charstring(seq):
    str_list = list()
    for i in range(len(seq)-4):
        str_list.append(seq[i:i+4])
    return str_list

In [268]:
populate4charstring('ATTAAAGGTTT')

['ATTA', 'TTAA', 'TAAA', 'AAAG', 'AAGG', 'AGGT', 'GGTT']

In [269]:
list_all_4char = list()
for row in range(DNAseq.shape[0]):
    list_all_4char = list_all_4char + populate4charstring(DNAseq.iloc[row][0]) 

In [270]:
len(list_all_4char)

28191

In [271]:
unique_4char = list(set(list_all_4char))

In [272]:
len(unique_4char)

256

In [273]:
dict_4char = {}
for sub_str in unique_4char:
    num = 0
    for s in list_all_4char:
        if sub_str == s:
            num += 1
    dict_4char[sub_str] = num

In [274]:
dict_4char 

{'CGGC': 19,
 'AGAT': 137,
 'TGCT': 240,
 'CCAG': 75,
 'CTCA': 118,
 'CGGA': 16,
 'AGCT': 138,
 'CGCC': 13,
 'TATG': 183,
 'CGCT': 27,
 'TTGG': 151,
 'TAAC': 125,
 'TGTG': 177,
 'CTGG': 97,
 'CATT': 151,
 'CAAA': 204,
 'TAGA': 141,
 'ACAT': 188,
 'ACGG': 28,
 'AGGC': 58,
 'AGTG': 136,
 'GGCG': 19,
 'TAGC': 75,
 'TACT': 193,
 'AGCG': 22,
 'AATC': 106,
 'GCAT': 77,
 'AGAG': 127,
 'GGTA': 117,
 'TTCG': 38,
 'TGGC': 106,
 'CGCA': 38,
 'TCTC': 74,
 'GAAA': 158,
 'AGTT': 168,
 'AAAC': 180,
 'ACCA': 138,
 'CTTC': 126,
 'ATGG': 167,
 'TATT': 196,
 'CCTT': 115,
 'TAAG': 93,
 'CCAT': 75,
 'AAGG': 112,
 'GTTC': 89,
 'TCCC': 24,
 'TCTA': 136,
 'TTAA': 240,
 'ATAA': 150,
 'TCGT': 41,
 'CCGC': 19,
 'CCGT': 26,
 'CCAA': 99,
 'GAAT': 115,
 'TTAG': 120,
 'ACCT': 135,
 'TAGT': 130,
 'TGCG': 31,
 'AGGT': 116,
 'GTGG': 106,
 'AACT': 190,
 'CTTA': 178,
 'CCTC': 54,
 'ATTT': 217,
 'CGTT': 43,
 'ATGC': 145,
 'CGAC': 14,
 'TTTG': 250,
 'AGAA': 192,
 'CATA': 105,
 'GGTC': 27,
 'ACTA': 162,
 'TTTT': 291,
 'TCAG

In [275]:
maxVal = -100
maxKey = None
for k in dict_4char.keys():
    if dict_4char[k] > maxVal:
        maxVal = dict_4char[k]
        maxKey = k

In [276]:
maxKey

'TGTT'

In [277]:
print(f'max 4-long-sequence is {maxKey} with frequency equals {maxKey}')

max 4-long-sequence is TGTT with frequency equals TGTT


## 3> จากไฟล์ data.xlsx สร้างตาราง
- ใหม่ที่มี 2 colum โดย colum แรกชื่อว่า ภาษาอีสาน และมีค่าใน colum เป็นค่า จาก colum ภาษาอีสานเดิมแต่เปลี่ยน คำว่าเป็น '/'เป็น',' และ colum ที่สองชื่อว่าภาษาไทย และมีค่าใน colum เป็นค่าจาก colum ภาษาไทยเดิมแต่เปลี่ยน ค่าที่เป็น '/'เป็น',' เช่น จาก สวัสดีฝสุผู้สุคน/เด้อ/จ้า สวัสดี/ทุกๆคน/นะ/คะ เป็น สวัสดี,สุผู้สุคน,เด้อ,จ้า สวัสดี,ทุกๆคน,นะ,คะ
- สร้างตาราง Dictionary โดยตารางประกอบประกอบด้วยสอง columns คือภาษาอีสาน และภาษาไทย โดยค่าใน  column ภาษาอีสาน คือคำในภาษาอีสาน และ columื ภาษาไทย คือแปลเป็นภาษากลางของภาษาอีสานใน colnn แรก (ใช้ค่าจากตาราง data.xlsx โดยคำที่อยู่ในลำดับเดียวกันของ columns ภาษาอีสานและภาษาไทยคือคำแปลของกันและกัน จำนวนคำของทั้งสอง column เท่ากัน - ถ้าจำนวนคำระหว่าง column ไม่เท่ากันให้ข้าม record นั้นไป)

- ใหม่ที่มี 2 colum โดย colum แรกชื่อว่า ภาษาอีสาน และมีค่าใน colum เป็นค่า จาก colum ภาษาอีสานเดิมแต่เปลี่ยน คำว่าเป็น '/'เป็น',' และ colum ที่สองชื่อว่าภาษาไทย และมีค่าใน colum เป็นค่าจาก colum ภาษาไทยเดิมแต่เปลี่ยน ค่าที่เป็น '/'เป็น',' เช่น จาก สวัสดีฝสุผู้สุคน/เด้อ/จ้า สวัสดี/ทุกๆคน/นะ/คะ เป็น สวัสดี,สุผู้สุคน,เด้อ,จ้า สวัสดี,ทุกๆคน,นะ,คะ

In [278]:
import pandas as pd

In [279]:
data1_df = pd.read_excel('C:\\Users\\N-PC\\Documents\\python\\python_dataviz\\data.xlsx',sheet_name='v1')

In [280]:
data1_df

No                                          ภาษาอีสาน  \
0    1.0                          สวัสดี/สุผู้สุคน/เด้อ/จ้า   
1    2.0       กิน/ข้าว/ก่อน/เด้อ/จ้า/ ไผ/สิ/ถ่า/เค้า/ดาวน์   
2    3.0  เค้า/ดาว/กะ/ขอ/ให้/เค้า/ดาว/อย่าง/มี/ความ/สุข/...   
3    4.0  นี่/จ้า/อา/หาร/เย็น/เป็น/ลา/ซาน/ย่า/กับ/ผัก/สด...   
4    5.0                     ขอ/ให้/เชิญ/แซ่บ/เด้อ/พี่/น้อง   
5    6.0         แมท/ธิว/บอก/ว่า/อี/แม่/อย่า/พึ่ง/หลับ/เด้อ   
6    7.0                                       เงียบ/แฮงจัด   
7    8.0          บ่/มี/ความ/รื่น/เริง/อี/หยัง/เลย/ข้าง/นอก   
8    9.0               มื้อ/นี้/สิ/เบิ่ง/พุ/ตอน/หก/ทุ่ม/จ่ะ   
9   10.0             ถ้า/ฝน/ตก/หิ/มะ/ตก/คึ/สิ/บ่/ได้/ออก/ไป   
10  11.0  ปี/ที่/แล้ว/ออก/ไป/เบิ่ง/เขา/จุด/พุ/มัน/เป็น/ฝ...   
11  12.0                        เพิ่น/ก้ม/แบบ/คน/เมา/นะ/จ้ะ   
12  13.0  ก้ม/ก้ม/ส่อง/พุ/เพิ่น/จุด/ละ/เพิ่น/กะ/ก้ม/ส่อง...   
13  14.0            เพิ่น/สิ/โซ/กล้าม/เพิ่น/หั่น/นะ/พี่น้อง   
14  15.0  โอ/เค/จ้า/สุผู้สุคน/มื้อ/นี้/จั่ง/ใด๋/กะ/ขอบ/ค...   
15  16.0  จ้า/พ้อ/กัน/ใหม่/ใน/คลิป/วิ/ดี/โอ/หน้า/สวัสดี/...   
16   NaN                                                NaN   

                                              ภาษาไทย     เริ่ม        จบ  \
0                               สวัสดี/ทุก ๆ คน/นะ/คะ  00:00:16  00:00:18   
1           กิน/ข้าว/ก่อน/นะ/จ้ะ/ใคร/จะ/รอ/เค้า/ดาวน์  00:02:14  00:02:18   
2   เค้า/ดาวน์/ก็/ขอ/ให้/เค้า/ดาน์/อย่าง/มี/ความ/ส...  00:02:21  00:02:25   
3   นี่/จ้า/อา/หาร/เย็น/เป็น/ลา/ซาน/ย่า/กับ/ผัก/สด...  00:02:33  00:02:37   
4                         ขอ/ให้/เชิญ/อร่อย/นะ/ทุก/คน  00:02:38  00:02:41   
5           แมท/ธิว/บอก/ว่า/คุณ/แม่/อย่า/พึ่ง/หลับ/นะ  00:03:08  00:03:08   
6                                         เงียบ/มาก ๆ  00:09:51  00:09:53   
7            ไม่/มี/ความ/รื่น/เริง/อะ/ไร/เลย/ด้าน/นอก  00:10:07  00:10:11   
8                  วัน/นี้/จะ/ดู/พุ/ตอน/หก/ทุ่ม/นะ/คะ  00:10:29  00:10:33   
9             ถ้า/ฝน/ตก/หิ/มะ/ตก/คง/จะ/ไม่/ได้/ออก/ไป  00:10:40  00:10:43   
10  ปี/ที่/แล้ว/ออก/ไป/ดู/เขา/จุด/พุ/มัน/เป็น/ฝน/ห...  00:10:52  00:10:56   
11                         เขา/ก้ม/แบบ/คน/เมา/อ่ะ/ค่ะ  00:11:09  00:11:12   
12  ก้ม/ก้ม/ส่อง/พุ/เขา/จุด/ละ/เขา/ก็/ก้ม/ส่อง/เขา...  00:11:14  00:11:18   
13              เขา/จะ/โชว์/กล้าม/เขา/นั่น/แหละ/ทุกคน  00:13:11  00:13:13   
14  โอ/เค/จ้า/ทุก ๆ  คน/วัน/นี้/ยัง/ไง/ก็/ขอบ/คุณ/...  00:15:23  00:15:26   
15  จ้า/เจอ/กัน/ใหม่/ใน/คลิป/วิ/ดี/โอ/หน้า/สวัสดี/...  00:15:30  00:15:33   
16                                                NaN       NaN       NaN   

    จบ-เริ่ม                                                G2P  
0   00:00:02  s-a-z^-1 w-a-t^-1 d-ii-z^-0 *s-u-z^-1 *ph-uu-z...  
1   00:00:04  k-i-n^-0 *kh-aa-w^-2 *k-@@-n^-1 *d-qq-z^-2 *c-...  
2   00:00:04  kh-a-w^-3 *d-aa-w^-0 *k-a-z^-1 *kh-@@-z^-4 *h-...  
3   00:00:04  n-ii-z^-2 *c-aa-z^-2 *z-aa-z^-0 *h-aa-n^-4 *j-...  
4   00:00:03  kh-@@-z^-4 *h-a-j^-2 *ch-qq-n^-0 *s-xx-p^-2 *d...  
5   00:00:00  m-xx-t^-1 *th-i-w^-0 *b-@@-k^-1 *w-aa-z^-2 *z-...  
6   00:00:02                  ng-iia-p^-2 *h-xx-ng^-0 *c-a-t^-1  
7   00:00:04  b-@@-z^-1 *m-ii-z^-0 *khw-aa-m^-0 *r-vv-n^-2 *...  
8   00:00:04  m-vv-z^-3 *n-ii-z^-3 *s-i-z^-1 *b-q-ng^-1 *ph-...  
9   00:00:03  th-aa-z^-2 *f-o-n^-4 *t-o-k^-1 *h-i-z^-1 *m-a-...  
10  00:00:04  p-ii-z^-0 *th-ii-z^-2 *l-xx-w^-3 *z-@@-k^-1 *p...  
11  00:00:03  ph-qq-n^-2 *k-o-m^-3 *b-xx-p^-1 *kh-o-n^-0 *m-...  
12  00:00:04  k-o-m^-3 *k-o-m^-3 *s-@-ng^-1 *ph-u-z^-3 *ph-q...  
13  00:00:02  ph-qq-n^-2 *s-i-z^-1 *s-oo-z^-0 *kl-aa-m^-2 *p...  
14  00:00:03  z-oo-z^-0 *kh-ee-z^-0 *c-aa-z^-2\n*s-u-z^-1 *p...  
15  00:00:03  c-aa-z^-2 *ph-@@-z^-3 *k-a-n^-0 *m-a-j^-1 *n-a...  
16  00:00:49                                                NaN

In [281]:
def splitstr(col1):
    co1 = col1.split('/')
    return co1

In [282]:
splitstr(data1_df.iloc[0,1])

['สวัสดี', 'สุผู้สุคน', 'เด้อ', 'จ้า']

In [283]:
splitstr(data1_df.iloc[0,2])

['สวัสดี', 'ทุก ๆ คน', 'นะ', 'คะ']

- สร้างตาราง Dictionary โดยตารางประกอบประกอบด้วยสอง columns คือภาษาอีสาน และภาษาไทย โดยค่าใน  column ภาษาอีสาน คือคำในภาษาอีสาน และ columื ภาษาไทย คือแปลเป็นภาษากลางของภาษาอีสานใน colnn แรก (ใช้ค่าจากตาราง data.xlsx โดยคำที่อยู่ในลำดับเดียวกันของ columns ภาษาอีสานและภาษาไทยคือคำแปลของกันและกัน จำนวนคำของทั้งสอง column เท่ากัน - ถ้าจำนวนคำระหว่าง column ไม่เท่ากันให้ข้าม record นั้นไป)

In [284]:
list_11 = []
list_22 = []
for i in range(0,7):
    list_11 += splitstr(data1_df.iloc[i,1])
    list_22 += splitstr(data1_df.iloc[i,2])

In [285]:
def google(col1,col2):
    str = f'{col1} แปลว่า {col2}'
    return str

In [286]:
a1 = []
for i in range(len(list_11)):
    a1+= [google(list_11[i],list_22[i])]

In [287]:
list_11[1]

'สุผู้สุคน'

In [288]:
google(list_11[1],list_11[1])

'สุผู้สุคน แปลว่า สุผู้สุคน'

In [289]:
Aa1 = set(a1)

In [290]:
Aa1

{' ไผ แปลว่า ใคร',
 'กะ แปลว่า ก็',
 'กับ แปลว่า กับ',
 'กิน แปลว่า กิน',
 'ก่อน แปลว่า ก่อน',
 'ขอ แปลว่า ขอ',
 'ข้าว แปลว่า ข้าว',
 'ความ แปลว่า ความ',
 'จ้ะ แปลว่า จ้ะ',
 'จ้า แปลว่า คะ',
 'จ้า แปลว่า จ้ะ',
 'จ้า แปลว่า จ้า',
 'ซาน แปลว่า ซาน',
 'ดาว แปลว่า ดาน์',
 'ดาว แปลว่า ดาวน์',
 'ดาวน์ แปลว่า ดาวน์',
 'ถ่า แปลว่า รอ',
 'ธิว แปลว่า ธิว',
 'นะ แปลว่า นะ',
 'นี่ แปลว่า นี่',
 'น้อง แปลว่า คน',
 'บอก แปลว่า บอก',
 'ผัก แปลว่า ผัก',
 'พี่ แปลว่า ทุก',
 'พึ่ง แปลว่า พึ่ง',
 'มี แปลว่า มี',
 'ย่า แปลว่า ย่า',
 'ลา แปลว่า ลา',
 'ว่า แปลว่า ว่า',
 'สด แปลว่า สด',
 'สวัสดี แปลว่า สวัสดี',
 'สิ แปลว่า จะ',
 'สุข แปลว่า สุข',
 'สุผู้สุคน แปลว่า ทุก ๆ คน',
 'หลับ แปลว่า หลับ',
 'หาร แปลว่า หาร',
 'อย่า แปลว่า อย่า',
 'อย่าง แปลว่า อย่าง',
 'อา แปลว่า อา',
 'อี แปลว่า คุณ',
 'เค้า แปลว่า เค้า',
 'เงียบ แปลว่า เงียบ',
 'เชิญ แปลว่า เชิญ',
 'เด้อ แปลว่า นะ',
 'เป็น แปลว่า เป็น',
 'เย็น แปลว่า เย็น',
 'แซ่บ แปลว่า อร่อย',
 'แมท แปลว่า แมท',
 'แม่ แปลว่า แม่',
 'แฮงจัด แปลว่า มาก ๆ',
 'ให้ แปล

In [291]:
import pandas as pd

In [292]:
data1_df = pd.read_excel('C:\\Users\\N-PC\\Documents\\python\\python_dataviz\\data.xlsx',sheet_name='v1')

In [293]:
data1_df

No                                          ภาษาอีสาน  \
0    1.0                          สวัสดี/สุผู้สุคน/เด้อ/จ้า   
1    2.0       กิน/ข้าว/ก่อน/เด้อ/จ้า/ ไผ/สิ/ถ่า/เค้า/ดาวน์   
2    3.0  เค้า/ดาว/กะ/ขอ/ให้/เค้า/ดาว/อย่าง/มี/ความ/สุข/...   
3    4.0  นี่/จ้า/อา/หาร/เย็น/เป็น/ลา/ซาน/ย่า/กับ/ผัก/สด...   
4    5.0                     ขอ/ให้/เชิญ/แซ่บ/เด้อ/พี่/น้อง   
5    6.0         แมท/ธิว/บอก/ว่า/อี/แม่/อย่า/พึ่ง/หลับ/เด้อ   
6    7.0                                       เงียบ/แฮงจัด   
7    8.0          บ่/มี/ความ/รื่น/เริง/อี/หยัง/เลย/ข้าง/นอก   
8    9.0               มื้อ/นี้/สิ/เบิ่ง/พุ/ตอน/หก/ทุ่ม/จ่ะ   
9   10.0             ถ้า/ฝน/ตก/หิ/มะ/ตก/คึ/สิ/บ่/ได้/ออก/ไป   
10  11.0  ปี/ที่/แล้ว/ออก/ไป/เบิ่ง/เขา/จุด/พุ/มัน/เป็น/ฝ...   
11  12.0                        เพิ่น/ก้ม/แบบ/คน/เมา/นะ/จ้ะ   
12  13.0  ก้ม/ก้ม/ส่อง/พุ/เพิ่น/จุด/ละ/เพิ่น/กะ/ก้ม/ส่อง...   
13  14.0            เพิ่น/สิ/โซ/กล้าม/เพิ่น/หั่น/นะ/พี่น้อง   
14  15.0  โอ/เค/จ้า/สุผู้สุคน/มื้อ/นี้/จั่ง/ใด๋/กะ/ขอบ/ค...   
15  16.0  จ้า/พ้อ/กัน/ใหม่/ใน/คลิป/วิ/ดี/โอ/หน้า/สวัสดี/...   
16   NaN                                                NaN   

                                              ภาษาไทย     เริ่ม        จบ  \
0                               สวัสดี/ทุก ๆ คน/นะ/คะ  00:00:16  00:00:18   
1           กิน/ข้าว/ก่อน/นะ/จ้ะ/ใคร/จะ/รอ/เค้า/ดาวน์  00:02:14  00:02:18   
2   เค้า/ดาวน์/ก็/ขอ/ให้/เค้า/ดาน์/อย่าง/มี/ความ/ส...  00:02:21  00:02:25   
3   นี่/จ้า/อา/หาร/เย็น/เป็น/ลา/ซาน/ย่า/กับ/ผัก/สด...  00:02:33  00:02:37   
4                         ขอ/ให้/เชิญ/อร่อย/นะ/ทุก/คน  00:02:38  00:02:41   
5           แมท/ธิว/บอก/ว่า/คุณ/แม่/อย่า/พึ่ง/หลับ/นะ  00:03:08  00:03:08   
6                                         เงียบ/มาก ๆ  00:09:51  00:09:53   
7            ไม่/มี/ความ/รื่น/เริง/อะ/ไร/เลย/ด้าน/นอก  00:10:07  00:10:11   
8                  วัน/นี้/จะ/ดู/พุ/ตอน/หก/ทุ่ม/นะ/คะ  00:10:29  00:10:33   
9             ถ้า/ฝน/ตก/หิ/มะ/ตก/คง/จะ/ไม่/ได้/ออก/ไป  00:10:40  00:10:43   
10  ปี/ที่/แล้ว/ออก/ไป/ดู/เขา/จุด/พุ/มัน/เป็น/ฝน/ห...  00:10:52  00:10:56   
11                         เขา/ก้ม/แบบ/คน/เมา/อ่ะ/ค่ะ  00:11:09  00:11:12   
12  ก้ม/ก้ม/ส่อง/พุ/เขา/จุด/ละ/เขา/ก็/ก้ม/ส่อง/เขา...  00:11:14  00:11:18   
13              เขา/จะ/โชว์/กล้าม/เขา/นั่น/แหละ/ทุกคน  00:13:11  00:13:13   
14  โอ/เค/จ้า/ทุก ๆ  คน/วัน/นี้/ยัง/ไง/ก็/ขอบ/คุณ/...  00:15:23  00:15:26   
15  จ้า/เจอ/กัน/ใหม่/ใน/คลิป/วิ/ดี/โอ/หน้า/สวัสดี/...  00:15:30  00:15:33   
16                                                NaN       NaN       NaN   

    จบ-เริ่ม                                                G2P  
0   00:00:02  s-a-z^-1 w-a-t^-1 d-ii-z^-0 *s-u-z^-1 *ph-uu-z...  
1   00:00:04  k-i-n^-0 *kh-aa-w^-2 *k-@@-n^-1 *d-qq-z^-2 *c-...  
2   00:00:04  kh-a-w^-3 *d-aa-w^-0 *k-a-z^-1 *kh-@@-z^-4 *h-...  
3   00:00:04  n-ii-z^-2 *c-aa-z^-2 *z-aa-z^-0 *h-aa-n^-4 *j-...  
4   00:00:03  kh-@@-z^-4 *h-a-j^-2 *ch-qq-n^-0 *s-xx-p^-2 *d...  
5   00:00:00  m-xx-t^-1 *th-i-w^-0 *b-@@-k^-1 *w-aa-z^-2 *z-...  
6   00:00:02                  ng-iia-p^-2 *h-xx-ng^-0 *c-a-t^-1  
7   00:00:04  b-@@-z^-1 *m-ii-z^-0 *khw-aa-m^-0 *r-vv-n^-2 *...  
8   00:00:04  m-vv-z^-3 *n-ii-z^-3 *s-i-z^-1 *b-q-ng^-1 *ph-...  
9   00:00:03  th-aa-z^-2 *f-o-n^-4 *t-o-k^-1 *h-i-z^-1 *m-a-...  
10  00:00:04  p-ii-z^-0 *th-ii-z^-2 *l-xx-w^-3 *z-@@-k^-1 *p...  
11  00:00:03  ph-qq-n^-2 *k-o-m^-3 *b-xx-p^-1 *kh-o-n^-0 *m-...  
12  00:00:04  k-o-m^-3 *k-o-m^-3 *s-@-ng^-1 *ph-u-z^-3 *ph-q...  
13  00:00:02  ph-qq-n^-2 *s-i-z^-1 *s-oo-z^-0 *kl-aa-m^-2 *p...  
14  00:00:03  z-oo-z^-0 *kh-ee-z^-0 *c-aa-z^-2\n*s-u-z^-1 *p...  
15  00:00:03  c-aa-z^-2 *ph-@@-z^-3 *k-a-n^-0 *m-a-j^-1 *n-a...  
16  00:00:49                                                NaN

In [294]:
google(list_11[0],list_22[0])

'สวัสดี แปลว่า สวัสดี'

In [295]:
google(list_11[1],list_22[1])

'สุผู้สุคน แปลว่า ทุก ๆ คน'

In [296]:
google(list_11[2],list_22[2])

'เด้อ แปลว่า นะ'

In [297]:
google(list_11[3],list_22[3])

'จ้า แปลว่า คะ'

In [298]:
def replacestr(col1):
    co1 = col1.replace("/", " ")
    return co1

In [299]:
replacestr(data1_df.iloc[0,1])

'สวัสดี สุผู้สุคน เด้อ จ้า'

In [300]:
data1_df = data1_df.dropna()

In [301]:
data1_df.iloc[0,1].replace("/", " ")

'สวัสดี สุผู้สุคน เด้อ จ้า'

In [345]:
data1_df['ภาษาอีสานใหม่'] = list_1
data1_df['ภาษาไทยใหม่'] = list_2

C:\Users\N-PC\Miniconda3\lib\site-packages\ipykernel_launcher.py:1: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  """Entry point for launching an IPython kernel.
C:\Users\N-PC\Miniconda3\lib\site-packages\ipykernel_launcher.py:2: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  


In [302]:
data1_df

No                                          ภาษาอีสาน  \
0    1.0                          สวัสดี/สุผู้สุคน/เด้อ/จ้า   
1    2.0       กิน/ข้าว/ก่อน/เด้อ/จ้า/ ไผ/สิ/ถ่า/เค้า/ดาวน์   
2    3.0  เค้า/ดาว/กะ/ขอ/ให้/เค้า/ดาว/อย่าง/มี/ความ/สุข/...   
3    4.0  นี่/จ้า/อา/หาร/เย็น/เป็น/ลา/ซาน/ย่า/กับ/ผัก/สด...   
4    5.0                     ขอ/ให้/เชิญ/แซ่บ/เด้อ/พี่/น้อง   
5    6.0         แมท/ธิว/บอก/ว่า/อี/แม่/อย่า/พึ่ง/หลับ/เด้อ   
6    7.0                                       เงียบ/แฮงจัด   
7    8.0          บ่/มี/ความ/รื่น/เริง/อี/หยัง/เลย/ข้าง/นอก   
8    9.0               มื้อ/นี้/สิ/เบิ่ง/พุ/ตอน/หก/ทุ่ม/จ่ะ   
9   10.0             ถ้า/ฝน/ตก/หิ/มะ/ตก/คึ/สิ/บ่/ได้/ออก/ไป   
10  11.0  ปี/ที่/แล้ว/ออก/ไป/เบิ่ง/เขา/จุด/พุ/มัน/เป็น/ฝ...   
11  12.0                        เพิ่น/ก้ม/แบบ/คน/เมา/นะ/จ้ะ   
12  13.0  ก้ม/ก้ม/ส่อง/พุ/เพิ่น/จุด/ละ/เพิ่น/กะ/ก้ม/ส่อง...   
13  14.0            เพิ่น/สิ/โซ/กล้าม/เพิ่น/หั่น/นะ/พี่น้อง   
14  15.0  โอ/เค/จ้า/สุผู้สุคน/มื้อ/นี้/จั่ง/ใด๋/กะ/ขอบ/ค...   
15  16.0  จ้า/พ้อ/กัน/ใหม่/ใน/คลิป/วิ/ดี/โอ/หน้า/สวัสดี/...   

                                              ภาษาไทย     เริ่ม        จบ  \
0                               สวัสดี/ทุก ๆ คน/นะ/คะ  00:00:16  00:00:18   
1           กิน/ข้าว/ก่อน/นะ/จ้ะ/ใคร/จะ/รอ/เค้า/ดาวน์  00:02:14  00:02:18   
2   เค้า/ดาวน์/ก็/ขอ/ให้/เค้า/ดาน์/อย่าง/มี/ความ/ส...  00:02:21  00:02:25   
3   นี่/จ้า/อา/หาร/เย็น/เป็น/ลา/ซาน/ย่า/กับ/ผัก/สด...  00:02:33  00:02:37   
4                         ขอ/ให้/เชิญ/อร่อย/นะ/ทุก/คน  00:02:38  00:02:41   
5           แมท/ธิว/บอก/ว่า/คุณ/แม่/อย่า/พึ่ง/หลับ/นะ  00:03:08  00:03:08   
6                                         เงียบ/มาก ๆ  00:09:51  00:09:53   
7            ไม่/มี/ความ/รื่น/เริง/อะ/ไร/เลย/ด้าน/นอก  00:10:07  00:10:11   
8                  วัน/นี้/จะ/ดู/พุ/ตอน/หก/ทุ่ม/นะ/คะ  00:10:29  00:10:33   
9             ถ้า/ฝน/ตก/หิ/มะ/ตก/คง/จะ/ไม่/ได้/ออก/ไป  00:10:40  00:10:43   
10  ปี/ที่/แล้ว/ออก/ไป/ดู/เขา/จุด/พุ/มัน/เป็น/ฝน/ห...  00:10:52  00:10:56   
11                         เขา/ก้ม/แบบ/คน/เมา/อ่ะ/ค่ะ  00:11:09  00:11:12   
12  ก้ม/ก้ม/ส่อง/พุ/เขา/จุด/ละ/เขา/ก็/ก้ม/ส่อง/เขา...  00:11:14  00:11:18   
13              เขา/จะ/โชว์/กล้าม/เขา/นั่น/แหละ/ทุกคน  00:13:11  00:13:13   
14  โอ/เค/จ้า/ทุก ๆ  คน/วัน/นี้/ยัง/ไง/ก็/ขอบ/คุณ/...  00:15:23  00:15:26   
15  จ้า/เจอ/กัน/ใหม่/ใน/คลิป/วิ/ดี/โอ/หน้า/สวัสดี/...  00:15:30  00:15:33   

    จบ-เริ่ม                                                G2P  
0   00:00:02  s-a-z^-1 w-a-t^-1 d-ii-z^-0 *s-u-z^-1 *ph-uu-z...  
1   00:00:04  k-i-n^-0 *kh-aa-w^-2 *k-@@-n^-1 *d-qq-z^-2 *c-...  
2   00:00:04  kh-a-w^-3 *d-aa-w^-0 *k-a-z^-1 *kh-@@-z^-4 *h-...  
3   00:00:04  n-ii-z^-2 *c-aa-z^-2 *z-aa-z^-0 *h-aa-n^-4 *j-...  
4   00:00:03  kh-@@-z^-4 *h-a-j^-2 *ch-qq-n^-0 *s-xx-p^-2 *d...  
5   00:00:00  m-xx-t^-1 *th-i-w^-0 *b-@@-k^-1 *w-aa-z^-2 *z-...  
6   00:00:02                  ng-iia-p^-2 *h-xx-ng^-0 *c-a-t^-1  
7   00:00:04  b-@@-z^-1 *m-ii-z^-0 *khw-aa-m^-0 *r-vv-n^-2 *...  
8   00:00:04  m-vv-z^-3 *n-ii-z^-3 *s-i-z^-1 *b-q-ng^-1 *ph-...  
9   00:00:03  th-aa-z^-2 *f-o-n^-4 *t-o-k^-1 *h-i-z^-1 *m-a-...  
10  00:00:04  p-ii-z^-0 *th-ii-z^-2 *l-xx-w^-3 *z-@@-k^-1 *p...  
11  00:00:03  ph-qq-n^-2 *k-o-m^-3 *b-xx-p^-1 *kh-o-n^-0 *m-...  
12  00:00:04  k-o-m^-3 *k-o-m^-3 *s-@-ng^-1 *ph-u-z^-3 *ph-q...  
13  00:00:02  ph-qq-n^-2 *s-i-z^-1 *s-oo-z^-0 *kl-aa-m^-2 *p...  
14  00:00:03  z-oo-z^-0 *kh-ee-z^-0 *c-aa-z^-2\n*s-u-z^-1 *p...  
15  00:00:03  c-aa-z^-2 *ph-@@-z^-3 *k-a-n^-0 *m-a-j^-1 *n-a...

In [303]:
def replacestr(col1):
    co1 = col1.replace("/", " ")
    return co1

In [304]:
replacestr(data1_df.iloc[0,1])

'สวัสดี สุผู้สุคน เด้อ จ้า'

In [305]:
list_1 = []
list_2 = []
for i in range(len(data1_df)):
    list_1.append(replacestr(data1_df.iloc[i,1]))
    list_2.append(replacestr(data1_df.iloc[i,2]))

In [306]:
list_1

['สวัสดี สุผู้สุคน เด้อ จ้า',
 'กิน ข้าว ก่อน เด้อ จ้า  ไผ สิ ถ่า เค้า ดาวน์',
 'เค้า ดาว กะ ขอ ให้ เค้า ดาว อย่าง มี ความ สุข นะ จ้ะ',
 'นี่ จ้า อา หาร เย็น เป็น ลา ซาน ย่า กับ ผัก สด เด้อ จ้า',
 'ขอ ให้ เชิญ แซ่บ เด้อ พี่ น้อง',
 'แมท ธิว บอก ว่า อี แม่ อย่า พึ่ง หลับ เด้อ',
 'เงียบ แฮงจัด',
 'บ่ มี ความ รื่น เริง อี หยัง เลย ข้าง นอก',
 'มื้อ นี้ สิ เบิ่ง พุ ตอน หก ทุ่ม จ่ะ',
 'ถ้า ฝน ตก หิ มะ ตก คึ สิ บ่ ได้ ออก ไป',
 'ปี ที่ แล้ว ออก ไป เบิ่ง เขา จุด พุ มัน เป็น ฝน หิ มะ นะ จ่ะ',
 'เพิ่น ก้ม แบบ คน เมา นะ จ้ะ',
 'ก้ม ก้ม ส่อง พุ เพิ่น จุด ละ เพิ่น กะ ก้ม ส่อง เพิ่น บ่ เงย หน้า ขึ้น หั่น นะ',
 'เพิ่น สิ โซ กล้าม เพิ่น หั่น นะ พี่น้อง',
 'โอ เค จ้า สุผู้สุคน มื้อ นี้ จั่ง ใด๋ กะ ขอบ คุณ สุผู้สุคน ที่ แวะ เวียน เข้า มา ทัก ทาย กับ ครัว คุณ สม บูรณ์',
 'จ้า พ้อ กัน ใหม่ ใน คลิป วิ ดี โอ หน้า สวัสดี ปี ใหม่ บ้าย บาย จ้า']

In [307]:
list_2

['สวัสดี ทุก ๆ คน นะ คะ',
 'กิน ข้าว ก่อน นะ จ้ะ ใคร จะ รอ เค้า ดาวน์',
 'เค้า ดาวน์ ก็ ขอ ให้ เค้า ดาน์ อย่าง มี ความ สุข นะ จ้ะ',
 'นี่ จ้า อา หาร เย็น เป็น ลา ซาน ย่า กับ ผัก สด นะ คะ',
 'ขอ ให้ เชิญ อร่อย นะ ทุก คน',
 'แมท ธิว บอก ว่า คุณ แม่ อย่า พึ่ง หลับ นะ',
 'เงียบ มาก ๆ',
 'ไม่ มี ความ รื่น เริง อะ ไร เลย ด้าน นอก',
 'วัน นี้ จะ ดู พุ ตอน หก ทุ่ม นะ คะ',
 'ถ้า ฝน ตก หิ มะ ตก คง จะ ไม่ ได้ ออก ไป',
 'ปี ที่ แล้ว ออก ไป ดู เขา จุด พุ มัน เป็น ฝน หิ มะ นะ คะ',
 'เขา ก้ม แบบ คน เมา อ่ะ ค่ะ',
 'ก้ม ก้ม ส่อง พุ เขา จุด ละ เขา ก็ ก้ม ส่อง เขา ไม่ แหงน หน้า ขึ้น นั่น แหละ',
 'เขา จะ โชว์ กล้าม เขา นั่น แหละ ทุกคน',
 'โอ เค จ้า ทุก ๆ  คน วัน นี้ ยัง ไง ก็ ขอบ คุณ ทุก ๆ  คน ที่ แวะ เวียน เข้า มา ทัก ทาย กับ ครัว คุณ สม บูรณ์',
 'จ้า เจอ กัน ใหม่ ใน คลิป วิ ดี โอ หน้า สวัสดี ปี ใหม่ บ้าย บาย จ้า']

ตารางใหม่ประกอบด้วยสองคอลัมและคำแปล คือภาษาอีสานใหม่และภาษาไทยใหม่

In [309]:
data1_df

No                                          ภาษาอีสาน  \
0    1.0                          สวัสดี/สุผู้สุคน/เด้อ/จ้า   
1    2.0       กิน/ข้าว/ก่อน/เด้อ/จ้า/ ไผ/สิ/ถ่า/เค้า/ดาวน์   
2    3.0  เค้า/ดาว/กะ/ขอ/ให้/เค้า/ดาว/อย่าง/มี/ความ/สุข/...   
3    4.0  นี่/จ้า/อา/หาร/เย็น/เป็น/ลา/ซาน/ย่า/กับ/ผัก/สด...   
4    5.0                     ขอ/ให้/เชิญ/แซ่บ/เด้อ/พี่/น้อง   
5    6.0         แมท/ธิว/บอก/ว่า/อี/แม่/อย่า/พึ่ง/หลับ/เด้อ   
6    7.0                                       เงียบ/แฮงจัด   
7    8.0          บ่/มี/ความ/รื่น/เริง/อี/หยัง/เลย/ข้าง/นอก   
8    9.0               มื้อ/นี้/สิ/เบิ่ง/พุ/ตอน/หก/ทุ่ม/จ่ะ   
9   10.0             ถ้า/ฝน/ตก/หิ/มะ/ตก/คึ/สิ/บ่/ได้/ออก/ไป   
10  11.0  ปี/ที่/แล้ว/ออก/ไป/เบิ่ง/เขา/จุด/พุ/มัน/เป็น/ฝ...   
11  12.0                        เพิ่น/ก้ม/แบบ/คน/เมา/นะ/จ้ะ   
12  13.0  ก้ม/ก้ม/ส่อง/พุ/เพิ่น/จุด/ละ/เพิ่น/กะ/ก้ม/ส่อง...   
13  14.0            เพิ่น/สิ/โซ/กล้าม/เพิ่น/หั่น/นะ/พี่น้อง   
14  15.0  โอ/เค/จ้า/สุผู้สุคน/มื้อ/นี้/จั่ง/ใด๋/กะ/ขอบ/ค...   
15  16.0  จ้า/พ้อ/กัน/ใหม่/ใน/คลิป/วิ/ดี/โอ/หน้า/สวัสดี/...   

                                              ภาษาไทย     เริ่ม        จบ  \
0                               สวัสดี/ทุก ๆ คน/นะ/คะ  00:00:16  00:00:18   
1           กิน/ข้าว/ก่อน/นะ/จ้ะ/ใคร/จะ/รอ/เค้า/ดาวน์  00:02:14  00:02:18   
2   เค้า/ดาวน์/ก็/ขอ/ให้/เค้า/ดาน์/อย่าง/มี/ความ/ส...  00:02:21  00:02:25   
3   นี่/จ้า/อา/หาร/เย็น/เป็น/ลา/ซาน/ย่า/กับ/ผัก/สด...  00:02:33  00:02:37   
4                         ขอ/ให้/เชิญ/อร่อย/นะ/ทุก/คน  00:02:38  00:02:41   
5           แมท/ธิว/บอก/ว่า/คุณ/แม่/อย่า/พึ่ง/หลับ/นะ  00:03:08  00:03:08   
6                                         เงียบ/มาก ๆ  00:09:51  00:09:53   
7            ไม่/มี/ความ/รื่น/เริง/อะ/ไร/เลย/ด้าน/นอก  00:10:07  00:10:11   
8                  วัน/นี้/จะ/ดู/พุ/ตอน/หก/ทุ่ม/นะ/คะ  00:10:29  00:10:33   
9             ถ้า/ฝน/ตก/หิ/มะ/ตก/คง/จะ/ไม่/ได้/ออก/ไป  00:10:40  00:10:43   
10  ปี/ที่/แล้ว/ออก/ไป/ดู/เขา/จุด/พุ/มัน/เป็น/ฝน/ห...  00:10:52  00:10:56   
11                         เขา/ก้ม/แบบ/คน/เมา/อ่ะ/ค่ะ  00:11:09  00:11:12   
12  ก้ม/ก้ม/ส่อง/พุ/เขา/จุด/ละ/เขา/ก็/ก้ม/ส่อง/เขา...  00:11:14  00:11:18   
13              เขา/จะ/โชว์/กล้าม/เขา/นั่น/แหละ/ทุกคน  00:13:11  00:13:13   
14  โอ/เค/จ้า/ทุก ๆ  คน/วัน/นี้/ยัง/ไง/ก็/ขอบ/คุณ/...  00:15:23  00:15:26   
15  จ้า/เจอ/กัน/ใหม่/ใน/คลิป/วิ/ดี/โอ/หน้า/สวัสดี/...  00:15:30  00:15:33   

    จบ-เริ่ม                                                G2P  \
0   00:00:02  s-a-z^-1 w-a-t^-1 d-ii-z^-0 *s-u-z^-1 *ph-uu-z...   
1   00:00:04  k-i-n^-0 *kh-aa-w^-2 *k-@@-n^-1 *d-qq-z^-2 *c-...   
2   00:00:04  kh-a-w^-3 *d-aa-w^-0 *k-a-z^-1 *kh-@@-z^-4 *h-...   
3   00:00:04  n-ii-z^-2 *c-aa-z^-2 *z-aa-z^-0 *h-aa-n^-4 *j-...   
4   00:00:03  kh-@@-z^-4 *h-a-j^-2 *ch-qq-n^-0 *s-xx-p^-2 *d...   
5   00:00:00  m-xx-t^-1 *th-i-w^-0 *b-@@-k^-1 *w-aa-z^-2 *z-...   
6   00:00:02                  ng-iia-p^-2 *h-xx-ng^-0 *c-a-t^-1   
7   00:00:04  b-@@-z^-1 *m-ii-z^-0 *khw-aa-m^-0 *r-vv-n^-2 *...   
8   00:00:04  m-vv-z^-3 *n-ii-z^-3 *s-i-z^-1 *b-q-ng^-1 *ph-...   
9   00:00:03  th-aa-z^-2 *f-o-n^-4 *t-o-k^-1 *h-i-z^-1 *m-a-...   
10  00:00:04  p-ii-z^-0 *th-ii-z^-2 *l-xx-w^-3 *z-@@-k^-1 *p...   
11  00:00:03  ph-qq-n^-2 *k-o-m^-3 *b-xx-p^-1 *kh-o-n^-0 *m-...   
12  00:00:04  k-o-m^-3 *k-o-m^-3 *s-@-ng^-1 *ph-u-z^-3 *ph-q...   
13  00:00:02  ph-qq-n^-2 *s-i-z^-1 *s-oo-z^-0 *kl-aa-m^-2 *p...   
14  00:00:03  z-oo-z^-0 *kh-ee-z^-0 *c-aa-z^-2\n*s-u-z^-1 *p...   
15  00:00:03  c-aa-z^-2 *ph-@@-z^-3 *k-a-n^-0 *m-a-j^-1 *n-a...   

                                        ภาษาอีสานใหม่  \
0                           สวัสดี สุผู้สุคน เด้อ จ้า   
1        กิน ข้าว ก่อน เด้อ จ้า  ไผ สิ ถ่า เค้า ดาวน์   
2   เค้า ดาว กะ ขอ ให้ เค้า ดาว อย่าง มี ความ สุข ...   
3   นี่ จ้า อา หาร เย็น เป็น ลา ซาน ย่า กับ ผัก สด...   
4                      ขอ ให้ เชิญ แซ่บ เด้อ พี่ น้อง   
5          แมท ธิว บอก ว่า อี แม่ อย่า พึ่ง หลับ เด้อ   
6                                        เงียบ แฮงจัด   
7           บ่ มี ความ รื่น 

In [310]:
import pandas as pd

In [311]:
data2_df = pd.read_excel('C:\\Users\\N-PC\\Documents\\python\\python_dataviz\\data.xlsx',sheet_name='v2')

In [312]:
data2_df 

No                                          ภาษาอีสาน  \
0    1.0                  สะ/หวัด/ดี/จ้า/สุผู้สุคน/เด้อ/จ้า   
1    2.0  อ้อ/ได้/รับ/อั่น/ไปรษณีย์/เนาะ/จ้า/ไปรษณีย์/เด...   
2    3.0  เออ/เพิ่น/เขียน/อั่น/ลุง/อั่น/ลุง/เขียน/อั่น/อ...   
3    4.0  ลุง/ลุง/ฮา/หลาด/เป็น/เป็น/สา/มี/ของ/พี่/สาว/เน...   
4    5.0  โอ/เค้/ละ/กะ/มา/ต่อ/อั่น/เปิด/อั่น/นี้/เนาะ/จ่...   
5    6.0  สุ/ภา/พร/หงส์/ทอง/จ่ะ/น้อง/โอ๋/เนาะ/จ่ะ/พี่น้อ...   
6    7.0  ถาม/ว่า/เอา/ไป/ได้/บ่/เอา/ไป/บ่/ได้/หึ้ย/จ่ะ/เ...   
7    8.0                 ไผ/ที่/ได้/เบิ่ง/คลิป/นี้/เนาะ/จ่ะ   
8    9.0  เดี๋ยว/ให้/เพิ่น/มา/เปิด/อีก/เทื่อ/หลัง/จ่ะ/เด...   
9   10.0  อั่น/นี้/คือ/คือ/คือ/น้า/น้อง/โอ๋/น้อง/โอ๋/เพิ...   
10  11.0  เพิ่น/กะ/เกรง/ใจ/เฮา/เฮา/กะ/เกรง/ใจ/เพิ่น/เด้จ...   
11  12.0  โอ้ย/ซื้อ/ของ/ถืก/ใจ/มา/ให้/เพิ่น/เด้/น้า/โอ๋/...   
12  13.0     อั่น/เพิ่น/มัก/ลิ/เวอร์/พูล/อยู่/แล้ว/หั่น/น้า   
13  14.0         นี่/เปิด/จั่ง/สี้/ต้อง/เอา/อัน/นี้/ออก/จ่ะ   
14  15.0  อัน/นี้/กะ/เป็น/ของ/อะ/เล็ก/แล้ว/นิ/อะ/เล็ก/เพ...   
15  16.0             โอ้ย/กะ/ขอบ/คุณ/สุผู้สุคน/หลายเด้อ/จ่ะ   
16  17.0  ถืก/ใจ/ถืก/ใจ/อ่า/ทุก/ทุก/อย่าง/ที่/ส่ง/มา/ให้...   
17  18.0  โอ/เค้/นั่น/หละ/ไผ/ที่/ติด/ตาม/แม่/บ้าน/ตลอด/ต...   
18  19.0  สำ/หรับ/อ่า/สำ/หรับ/แฟน/คลับ/เนาะ/จ้า/งวด/นี้/...   
19  20.0  อ่า/มา/เมือง/ไทย/งวด/นี้/อ่า/เฮา/ได้/หนึ่ง/แสน...   
20  21.0  มี/ของ/ราง/วัล/แจก/แฟน/คลับ/เนาะ/จ่ะ/แต่/กะ/ตา...   
21  22.0  ให่/ให่/ลง/ซื่อ/ให่/ลง/ซื่อ/แล้ว/กะ/เดี๋ยว/เฮา...   
22  23.0  ให่/ทุก/คน/ฮุ้/ว่า/ฮู้/ว่า/ใน/คลิป/วี/ดี้/โอ/น...   
23  24.0  อาจ/สิ/บ่/เทื่อ/ทั่ว/ถึง/เพราะ/ว่า/เฮา/มา/เมือ...   
24  25.0  ละ/กะ/เป้น/แฟน/คลับ/ที่/อ่า/ส่ง/ของ/ราง/วัล/มา...   
25  26.0  เฮ็ด/จั่ง/ใด๋/สิ/ได้/ราง/วัล/เนาะ/จ่ะ/แต่/ว่า/...   
26  27.0  มื้อ/นี้/ขอบ/คุณ/อ่า/ผู้/ที่/ซ่ง/ซ่ง/อ่า/ซ่ง/ส...   
27  28.0  ลุง/กะ/เป็น/เอ่อ/สา/มี/ของ/พี่/สาว/เนาะ/จ่ะ/เพ...   
28   NaN                                                NaN   

                                              ภาษาไทย     เริ่ม        จบ  \
0                       สะ/หวัด/ดี/จ้า/ทุก ๆ คน/นะ/คะ  00:02:51  00:02:53   
1   อ้อ/ได้/รับ/เอิ่ม/ไปรษณีย์/นะ/คะ/ไปรษณีย์/เดี๋...  00:03:07  00:03:12   
2   เอ่อ/เขา/เขียน/เอิ่ม/ลุง/เอิ่ม/ลุง/เขียน/เอิ่ม...  00:03:39  00:03:45   
3   ลุง/ลุง/ฮา/หลาด/เป็น/เป็น/สา/มี/ของ/พี่/สาว/นะ...  00:04:48  00:04:53   
4   โอ/เค/แล้ว/ก็/มา/ต่อ/เอิ่ม/เปิด/อัน/นี้/นะ/คะ/...  00:04:54  00:05:03   
5   สุ/ภา/พร/หงส์/ทอง/จ้า/น้อง/โอ๋/เนาะ/จ้า/ทุกคน/...  00:05:07  00:05:16   
6   ถาม/ว่า/เอา/ไป/ได้/ไหม/เอา/ไป/ไม่/ได้/หึ้ย/จ้า...  00:06:04  00:06:12   
7                       ใคร/ที่/ได้/ดู/คลิป/นี้/นะ/คะ  00:06:55  00:06:58   
8   เดี๋ยว/ให้/เขา/มา/เปิด/อีก/ที/หลัง/จ้า/เดี๋ยว/ให้  00:09:09  00:09:11   
9   อัน/นี้/คือ/คือ/คือ/น้า/น้อง/โอ๋/น้อง/โอ๋/เขา/...  00:09:13  00:09:27   
10  เขา/ก็/เกรง/ใจ/เรา/เรา/ก็/เกรง/ใจ/เขา/นั่นแหละ...  00:09:27  00:09:36   
11  โอ้ย/ซื้อ/ของ/ถูก/ใจ/มา/ให้/เขา/เนาะ/น้า/โอ๋/เอ้ย  00:10:10  00:10:13   
12     เอิ่ม/เขา/ชอบ/ลิ/เวอร์/พูล/อยู่/แล้ว/นั่น/แหละ  00:10:14  00:10:16   
13        นี่/เปิด/อย่าง/นี้/ต้อง/เอา/อัน/นี้/ออก/จ้า  00:11:19  00:11:22   
14  อัน/นี้/ก็/เป็น/ของ/อะ/เล็ก/แล้ว/แหละ/อะ/เล็ก/...  00:12:48  00:12:53   
15                โอ้ย/ก็/ขอบ/คุณ/ทุก ๆ คน/มาก ๆ /จ้า  00:12:54  00:12:56   
16  ถูก/ใจ/ถูก/ใจ/อ่า/ทุก/ทุก/อย่าง/ที่/ส่ง/มา/ให้...  00:13:00  00:13:09   
17  โอ/เค/นั่น/แหละ/ใคร/ที่/ติด/ตาม/แม่/บ้าน/ตลอด/...  00:13:11  00:13:23   
18  สำ/หรับ/อ่า/สำ/หรับ/แฟน/คลับ/นะ/คะ/งวด/นี้/อาจ...  00:13:23  00:13:30   
19  อ่า/มา/เมือง/ไทย/งวด/นี้/อ่า/เรา/ได้/หนึ่ง/แสน...  00:13:33  00:13:47   
20  มี/ของ/ราง/วัล/แจก/แฟน/คลับ/นะ/คะ/แต่/ก็/ตาม/น...  00:13:58  00:14:08   
21  ให้/ให้/ลง/ชื้อ/ให้/ลง/ชื่อ/แล้ว/ก็/เดี๋ยว/เรา...  00:14:09  00:14:16   
22  ให้/ทุก/คน/รู้/ว่า/รู้/ว่า/ใน/คลิป/วิ/ดี/โอ/นี...  00:14:17  00:14:30   
23  อาจ/จะ/ไม่/ทั่ว/ทั่ว/ถึง/เพราะ/ว่า/เรา/มา/เมือ...  00:14:30  00:14:44   
24  และ/ก็/เป็น/แฟน/คลับ/ที่/อ่า/ส่ง/ของ/ราง/วัล/ม...  00:14:45  00:14:57   
25  ทำ/ยัง/ไง/จะ/ได้/ราง/วัล/นะ/คะ/แต่/ว่า/คลิป/นี...  00:15:00  00:15:11   
26  วัน/นี้/ขอบ/คุณ/อ่า/ผู้/ที่/ส่ง/

In [313]:
def splitstr(col1):
    co1 = col1.split('/')
    return co1

In [314]:
splitstr(data2_df.iloc[0,1])

['สะ', 'หวัด', 'ดี', 'จ้า', 'สุผู้สุคน', 'เด้อ', 'จ้า']

In [315]:
splitstr(data2_df.iloc[0,2])

['สะ', 'หวัด', 'ดี', 'จ้า', 'ทุก ๆ คน', 'นะ', 'คะ']

In [316]:
list_11 = []
list_22 = []
for i in range(0,7):
    list_11 += splitstr(data2_df.iloc[i,1])
    list_22 += splitstr(data2_df.iloc[i,2])

In [317]:
def google(col1,col2):
    str = f'{col1} แปลว่า {col2}'
    return str

In [318]:
a1 = []
for i in range(len(list_11)):
    a1+= [google(list_11[i],list_22[i])]

In [319]:
list_11[1]

'หวัด'

In [320]:
google(list_11[1],list_11[1])

'หวัด แปลว่า หวัด'

In [321]:
Aa1 = set(a1)

In [322]:
Aa1

{'กะ แปลว่า ก็',
 'ของ แปลว่า ของ',
 'คริส แปลว่า คริส',
 'จาก แปลว่า จาก',
 'จ่ะ แปลว่า คะ',
 'จ่ะ แปลว่า จ้า',
 'จ้า แปลว่า คะ',
 'จ้า แปลว่า จ้า',
 'ซึ่ง แปลว่า ซึ่ง',
 'ซื้อ แปลว่า ซื้อ',
 'ดี แปลว่า ดี',
 'ต่อ แปลว่า ต่อ',
 'ถาม แปลว่า ถาม',
 'ทอง แปลว่า ทอง',
 'นั่น แปลว่า นั่น',
 'นี้ แปลว่า นี้',
 'น้อง แปลว่า น้อง',
 'น้า แปลว่า แหละ',
 'บอก แปลว่า บอก',
 'บัก แปลว่า ตั้ง',
 'บ็อค แปลว่า บ็อค',
 'บ่ แปลว่า ไม่',
 'บ่ แปลว่า ไหม',
 'ป้า แปลว่า ป้า',
 'พร แปลว่า พร',
 'พัสดุ แปลว่า พัสดุ',
 'พี่ แปลว่า พี่',
 'พี่น้อง แปลว่า ทุกคน',
 'ภา แปลว่า ภา',
 'มัด แปลว่า มัท',
 'มา แปลว่า มา',
 'มี แปลว่า มี',
 'รับ แปลว่า รับ',
 'ละ แปลว่า แล้ว',
 'ลุง แปลว่า ลุง',
 'ล่ะ แปลว่า แหละ',
 'ว่า แปลว่า ว่า',
 'สะ แปลว่า สะ',
 'สา แปลว่า สา',
 'สาว แปลว่า สาว',
 'สิ แปลว่า จะ',
 'สุ แปลว่า สุ',
 'สุผู้สุคน แปลว่า ทุก ๆ คน',
 'ส่ง แปลว่า ส่ง',
 'หงส์ แปลว่า หงส์',
 'หละ แปลว่า แหละ',
 'หลาด แปลว่า หลาด',
 'หล้าย แปลว่า เยอะ',
 'หวัด แปลว่า หวัด',
 'หึ้ย แปลว่า หึ้ย',
 'อวย แปลว่า อวย',
 'อัน แ

In [346]:
import pandas as pd

In [347]:
data3_df = pd.read_excel('C:\\Users\\N-PC\\Documents\\python\\python_dataviz\\data.xlsx',sheet_name='v3')

In [348]:
data3_df

No                                          ภาษาอีสาน  \
0    1.0         มื้อ/นี้/หนาว/แฮง/ต้อง/ไป่/ธุระ/โอ้ย/อั่น/   
1    2.0  หิ/มะ/บ่/ได้/ตก/ดอก/จ่ะ/แต่/ว่า/เป็น/น้ำ/แข็ง/...   
2    3.0  น้ำ/แข็ง/เกาะ/น้ำ/แข็ง/เกาะ/ตั้ง/แต่/มื้อ/คื้น...   
3    4.0  วื้ย/บ่/อยาก/ออก/ข้าง/นอก/เลย/คั้ก/แต่/ว่า/อั่...   
4    5.0  มือ/แข็ง/เลย/บ่/ได้/เอา/บ่/ได้/เอา/ถุง/มือ/มา/...   
5    6.0  อั่น/นี้/กะ/คือ/นั่น/หละ/ว่า/สิ/ลา/ท่าน/ผู้/ชม...   
6    7.0  โอ้ย/อา/กาศ/หนาว/แฮง/พี่น้อง/เพิ่ง/นี่/อั่น/เข...   
7    8.0  แต่/ว่า/แดด/ออก/เพิ่น/กะ/สิ/เอ่อ/ดี/ขึ้น/เนาะ/...   
8    9.0  โอ้ย/สิ/บอก/อี/หยัง/พี่น้อง/สวัสดี/โอ้ย/สวัสดี...   
9   10.0  อั่น/มัน/กะ/เหลือ/หาง/มัน/เนาะ/จ่ะ/ประ/มาณ/นี้...   
10  11.0  กะ/ว่า/อั่น/พ่อ/บ้าน/กลับ/มา/จาก/นอเวย์/ละ/กะ/...   
11  12.0  ที่/ไหน/ได้/นาง/บ่/กิน/ข้าว/แลง/กะ/คึด/ว่า/ปลา...   
12  13.0  กะ/ว่า/เอ่อ/เอ่อ/พ่อ/บ้าน/กิน/คั่น/ว่า/เพิ่น/ก...   
13  14.0  กะ/เลย/ได้/ข้าว/อั่น/ข้าว/ไข่/เจียว/เด้อ/พี่น้...   
14  15.0  ปลา/แห้ง/ทอด/แล้ว/กะ/เป็น/น้ำ/จิ้ม/อ่า/น้ำ/จิ้...   
15  16.0  เดี๋ยว/เดี๋ยว/เดี๋ยว/เดี๋ยว/สิ/ไห่/เบิ่ง/นี่/โ...   
16  17.0  ข่อย/กะ/เลย/สั่ง/เอา/หั่น/น้า/มา/แกะ/ถุง/เดี๋ย...   
17  18.0  อัน/นี้/ร้าน/กี/วี/เด้อ/พี่น้อง/เป็น/ร้าน/ซุปเ...   
18  19.0  นี่/พี่น้อง/พ่อ/บ้าน/ซื้อ/ปลา/ลัฟ/มา/ให้/ปลา/ล...   
19  20.0  ซื้อ/มา/ให้/สี่/แพ็ค/พี่น้อง/เมื่อ/ก่อน/ตอน/อย...   
20  21.0  มัน/แซ/แข็ง/มา/เด้อ/จ่ะ/ซื้อ/มา/จาก/กี/วี/นอเว...   
21  22.0  แต่/ว่า/กะ/ดี/ใจ/ได้/ของ/ฝาก/หมาย/ถึง/กะ/เฮา/บ...   
22  23.0  ข่อย/กะ/เลย/ว่า/เอา/กะ/อ้าว/กะ/เลย/ว่า/จั่ง/สี...   
23  24.0  ห้า/สิบ/กว่า/โครน/เพิ่น/ว่า/จั่ง/สี้/แต่/ว่า/บ...   
24  25.0  โอ/เค้/เอา/ไป/เก็บ/เอา/ไป/เก็บ/อัน/นี่/แซ่/แซ่...   
25  26.0  รวม/กัน/ตลอด/เลย/จ่ะ/คือ/เก่า/กิน/เลย/จ่ะ/นั่น...   
26  27.0  เมื่อ/วาน/อั่น/นอเวย์/ลบ/สิบ/อง/ศา/หม่อง/ที่/ค...   
27  28.0  เศรษฐกิจ/ของ/เพิ่น/เพิ่น/สิ/มี/อ่า/ปลา/แซล/มอล...   
28  29.0  ส่ง/ออก/นอก/ประ/เทศ/เนี่ย/ซื้อ/สม/มุติ/ว่า/ซื้...   
29  30.0  บาง/คน/ถาม/มา/ใน/คอม/เมนท์/เนาะ/จ่ะ/ว่า/รส/ชาต...   
30  31.0  ไผ/สิ/มัก/กิน/ซู/ชิ/ซึ่ง/เขา/มี/ใน/ประ/กอบ/อา/...   
31  32.0  ปลา/แซล/มอล/เฮ็ด/แกง/อื้อ/ขอ/โทษ/เฮ็ด/แกง/เขีย...   
32  33.0  พ่อ/บ้าน/กลับ/มา/นอน/ตั้ง/แต่/สอง/ทุ่ง/เคิ่ง/จ...   
33  34.0  ตั้ง/แต่/ฮู้/จัก/พ่อ/บ้าน/มา/ตั้ง/แต่/มา/อยู่/...   
34  35.0  คือ/เจ็บ/ปวด/หัว/ตัว/ร้อน/จั่ง/สี้/เป็น/ไข้/บ่...   
35  36.0  เพิ่น/นั่ง/ข้าง/ข้าง/หมู่/เป็น/คน/ขับ/อั่น/ขา/...   
36  37.0  นั่น/หละ/เป็น/อุบัติเหตุ/แรก/ที่/เรา/เฮ็ด/มา/ง...   
37  38.0  ได้/พัก/สั่ม/นั้น/เอง/ตั้ง/แต่/ฮู้/จัก/กัน/มา/...   
38  39.0  กะ/คือ/กะ/บ่/ได้/เจ็บ/ป่วย/ด้วย/ด้วย/ตัวเอง/หน...   
39  40.0  เดี๋ยว/กะ/สิ/กิน/ไป/แต่/ว่า/กะ/สิ/ลา/ท่าน/ผู้/...   
40  41.0  นั่น/หละ/สุผู้สุคน/เดี๋ยว/กะ/น้อง/แอน/ขอ/ขอ/อน...   
41   NaN                                                NaN   

                                              ภาษาไทย     เริ่ม        จบ  \
0           วัน/นี้/หนาว/มาก/ต้อง/ไป/ธุระ/โอ้ย/เอิ่ม/  00:00:05  00:00:11   
1   หิ/มะ/ไม่/ได้/ตก/หรอก/จ้า/แต่/ว่า/เป็น/น้ำ/แข็...  00:00:14  00:00:20   
2   น้ำ/แข็ง/เกาะ/น้ำ/แข็ง/เกาะ/ตั้ง/แต่/มื้อ/คื้น...  00:00:22  00:00:32   
3   ว้าย/ไม่/อยาก/ออก/ข้าง/นอก/เลย/จริง/แต่/ว่า/เอ...  00:00:33  00:00:46   
4   มือ/แข็ง/เลย/ไม่/ได้/เอา/ไม่/ได้/เอา/ถุง/มือ/ม...  00:00:48  00:00:54   
5   เอิ่ม/นี้/ก็/คือ/นั่น/แหละ/ว่า/จะ/ลา/ท่าน/ผู้/...  00:00:58  00:01:16   
6   โอ้ย/อา/กาศ/หนาว/มาก/ทุกคน/พึ่ง/นี่/เอิ่ม/เข้า...  00:01:27  00:01:38   
7   แต่/ว่า/แดด/ออก/เขา/ก็/จะ/เอ่อ/ดี/ขึ้น/นะ/คะ/ก...  00:01:39  00:01:53   
8   โอ้ย/จะ/บอก/อะ/ไร/ทุกคน/สวัสดี/โอ้ย/สวัสดี/หรื...  00:01:53  00:02:08   
9   เอิ่ม/มัน/ก็/เหลือ/หาง/มัน/นะ/คะ/ประ/มาณ/นี้/แ...  00:02:10  00:02:22   
10  ก็/ว่า/เอิ่ม/พ่อ/บ้าน/กลับ/มา/จาก/นอเวย์/แล้ว/...  00:02:23  00:02:46   
11  ที่/ไหน/ได้/นาง/ไม่/กิน/ข้าว/เย็น/ก็/คิด/ว่า/ป...  00:02:47  00:03:04   
12  ก็/ว่า/เอ่อ/เอ่อ/พ่อ/บ้าน/กิน/ถ้า/ว่า/เขา/กิน/...  00:03:06  00:03:21   
13  ก็/เลย/ได้/ข้าว/เอิ่ม/ข้าว/ไข่/เจียว/นะ/ทุกคน/...  00:03:27  00:03:31   
14  ปลา/แห้ง/ทอด/แล้ว/ก็/เป็น/น้ำ/จิ้ม/อ่า/น้ำ/จิ้...  00:03:38  00:03:54   
15  เดี๋ยว/เดี๋ยว/เดี๋ยว/เดี๋ยว/จะ/ให้/ดู/นี่/ถุ/ห...  00:04:05 

In [349]:
def splitstr(col1):
    co1 = col1.split('/')
    return co1

In [350]:
splitstr(data3_df.iloc[0,1])

['มื้อ', 'นี้', 'หนาว', 'แฮง', 'ต้อง', 'ไป่', 'ธุระ', 'โอ้ย', 'อั่น', '']

In [351]:
splitstr(data3_df.iloc[0,2])

['วัน', 'นี้', 'หนาว', 'มาก', 'ต้อง', 'ไป', 'ธุระ', 'โอ้ย', 'เอิ่ม', '']

In [352]:
list_11 = []
list_22 = []
for i in range(0,7):
    list_11 += splitstr(data3_df.iloc[i,1])
    list_22 += splitstr(data3_df.iloc[i,2])

In [353]:
def google(col1,col2):
    str = f'{col1} แปลว่า {col2}'
    return str

In [354]:
a1 = []
for i in range(len(list_11)):
    a1+= [google(list_11[i],list_22[i])]

In [355]:
list_11[1]

'นี้'

In [356]:
google(list_11[1],list_11[1])

'นี้ แปลว่า นี้'

In [357]:
Aa1 = set(a1)

In [358]:
Aa1

{' แปลว่า ',
 'กล้อง แปลว่า กล้อง',
 'กะ แปลว่า ก็',
 'กัน แปลว่า กัน',
 'กับ แปลว่า กับ',
 'กาศ แปลว่า กาศ',
 'ก่อน แปลว่า ก่อน',
 'ขอ แปลว่า ขอ',
 'ขาว แปลว่า ขาว',
 'ข้าง แปลว่า ข้าง',
 'คั้ก แปลว่า จริง',
 'คึด แปลว่า คิด',
 'คือ แปลว่า คือ',
 'คื้น แปลว่า คื้น',
 'คุย แปลว่า คุย',
 'จั่ง แปลว่า ยัง',
 'จั่ง แปลว่า อย่าง',
 'จ่ะ แปลว่า คะ',
 'จ่ะ แปลว่า จ้ะ',
 'จ่ะ แปลว่า จ้า',
 'ชม แปลว่า ชม',
 'ดอก แปลว่า หรอก',
 'ตก แปลว่า ตก',
 'ตั้ง แปลว่า ตั้ง',
 'ต้อง แปลว่า ต้อง',
 'ถุง แปลว่า ถุง',
 'ทัน แปลว่า ไม่',
 'ท่าน แปลว่า ท่าน',
 'ธุระ แปลว่า ธุระ',
 'นอก แปลว่า นอก',
 'นะ แปลว่า นะ',
 'นั่น แปลว่า นั่น',
 'นั้น แปลว่า นั้น',
 'นี่ แปลว่า นี่',
 'นี้ แปลว่า นี้',
 'นึง แปลว่า นึง',
 'น้ำ แปลว่า น้ำ',
 'บอก แปลว่า บอก',
 'บ่ แปลว่า ยัง',
 'บ่ แปลว่า ไม่',
 'บ้าน แปลว่า บ้าน',
 'ปิด แปลว่า ปิด',
 'ผู้ แปลว่า ผู้',
 'พฤหัส แปลว่า พฤหัส',
 'พี่น้อง แปลว่า ทุกคน',
 'มะ แปลว่า มะ',
 'มัน แปลว่า มัน',
 'มา แปลว่า มา',
 'มือ แปลว่า มือ',
 'มื้อ แปลว่า มื้อ',
 'มื้อ แปลว่า วัน',
 'ลบ แปลว่

In [359]:
data3_df = data3_df.dropna()

In [360]:
data3_df.iloc[0,1].replace("/", " ")

'มื้อ นี้ หนาว แฮง ต้อง ไป่ ธุระ โอ้ย อั่น '

In [361]:
data3_df

No                                          ภาษาอีสาน  \
0    1.0         มื้อ/นี้/หนาว/แฮง/ต้อง/ไป่/ธุระ/โอ้ย/อั่น/   
1    2.0  หิ/มะ/บ่/ได้/ตก/ดอก/จ่ะ/แต่/ว่า/เป็น/น้ำ/แข็ง/...   
2    3.0  น้ำ/แข็ง/เกาะ/น้ำ/แข็ง/เกาะ/ตั้ง/แต่/มื้อ/คื้น...   
3    4.0  วื้ย/บ่/อยาก/ออก/ข้าง/นอก/เลย/คั้ก/แต่/ว่า/อั่...   
4    5.0  มือ/แข็ง/เลย/บ่/ได้/เอา/บ่/ได้/เอา/ถุง/มือ/มา/...   
5    6.0  อั่น/นี้/กะ/คือ/นั่น/หละ/ว่า/สิ/ลา/ท่าน/ผู้/ชม...   
6    7.0  โอ้ย/อา/กาศ/หนาว/แฮง/พี่น้อง/เพิ่ง/นี่/อั่น/เข...   
7    8.0  แต่/ว่า/แดด/ออก/เพิ่น/กะ/สิ/เอ่อ/ดี/ขึ้น/เนาะ/...   
8    9.0  โอ้ย/สิ/บอก/อี/หยัง/พี่น้อง/สวัสดี/โอ้ย/สวัสดี...   
9   10.0  อั่น/มัน/กะ/เหลือ/หาง/มัน/เนาะ/จ่ะ/ประ/มาณ/นี้...   
10  11.0  กะ/ว่า/อั่น/พ่อ/บ้าน/กลับ/มา/จาก/นอเวย์/ละ/กะ/...   
11  12.0  ที่/ไหน/ได้/นาง/บ่/กิน/ข้าว/แลง/กะ/คึด/ว่า/ปลา...   
12  13.0  กะ/ว่า/เอ่อ/เอ่อ/พ่อ/บ้าน/กิน/คั่น/ว่า/เพิ่น/ก...   
13  14.0  กะ/เลย/ได้/ข้าว/อั่น/ข้าว/ไข่/เจียว/เด้อ/พี่น้...   
14  15.0  ปลา/แห้ง/ทอด/แล้ว/กะ/เป็น/น้ำ/จิ้ม/อ่า/น้ำ/จิ้...   
15  16.0  เดี๋ยว/เดี๋ยว/เดี๋ยว/เดี๋ยว/สิ/ไห่/เบิ่ง/นี่/โ...   
16  17.0  ข่อย/กะ/เลย/สั่ง/เอา/หั่น/น้า/มา/แกะ/ถุง/เดี๋ย...   
17  18.0  อัน/นี้/ร้าน/กี/วี/เด้อ/พี่น้อง/เป็น/ร้าน/ซุปเ...   
18  19.0  นี่/พี่น้อง/พ่อ/บ้าน/ซื้อ/ปลา/ลัฟ/มา/ให้/ปลา/ล...   
19  20.0  ซื้อ/มา/ให้/สี่/แพ็ค/พี่น้อง/เมื่อ/ก่อน/ตอน/อย...   
20  21.0  มัน/แซ/แข็ง/มา/เด้อ/จ่ะ/ซื้อ/มา/จาก/กี/วี/นอเว...   
21  22.0  แต่/ว่า/กะ/ดี/ใจ/ได้/ของ/ฝาก/หมาย/ถึง/กะ/เฮา/บ...   
22  23.0  ข่อย/กะ/เลย/ว่า/เอา/กะ/อ้าว/กะ/เลย/ว่า/จั่ง/สี...   
23  24.0  ห้า/สิบ/กว่า/โครน/เพิ่น/ว่า/จั่ง/สี้/แต่/ว่า/บ...   
24  25.0  โอ/เค้/เอา/ไป/เก็บ/เอา/ไป/เก็บ/อัน/นี่/แซ่/แซ่...   
25  26.0  รวม/กัน/ตลอด/เลย/จ่ะ/คือ/เก่า/กิน/เลย/จ่ะ/นั่น...   
26  27.0  เมื่อ/วาน/อั่น/นอเวย์/ลบ/สิบ/อง/ศา/หม่อง/ที่/ค...   
27  28.0  เศรษฐกิจ/ของ/เพิ่น/เพิ่น/สิ/มี/อ่า/ปลา/แซล/มอล...   
28  29.0  ส่ง/ออก/นอก/ประ/เทศ/เนี่ย/ซื้อ/สม/มุติ/ว่า/ซื้...   
29  30.0  บาง/คน/ถาม/มา/ใน/คอม/เมนท์/เนาะ/จ่ะ/ว่า/รส/ชาต...   
30  31.0  ไผ/สิ/มัก/กิน/ซู/ชิ/ซึ่ง/เขา/มี/ใน/ประ/กอบ/อา/...   
31  32.0  ปลา/แซล/มอล/เฮ็ด/แกง/อื้อ/ขอ/โทษ/เฮ็ด/แกง/เขีย...   
32  33.0  พ่อ/บ้าน/กลับ/มา/นอน/ตั้ง/แต่/สอง/ทุ่ง/เคิ่ง/จ...   
33  34.0  ตั้ง/แต่/ฮู้/จัก/พ่อ/บ้าน/มา/ตั้ง/แต่/มา/อยู่/...   
34  35.0  คือ/เจ็บ/ปวด/หัว/ตัว/ร้อน/จั่ง/สี้/เป็น/ไข้/บ่...   
35  36.0  เพิ่น/นั่ง/ข้าง/ข้าง/หมู่/เป็น/คน/ขับ/อั่น/ขา/...   
36  37.0  นั่น/หละ/เป็น/อุบัติเหตุ/แรก/ที่/เรา/เฮ็ด/มา/ง...   
37  38.0  ได้/พัก/สั่ม/นั้น/เอง/ตั้ง/แต่/ฮู้/จัก/กัน/มา/...   
38  39.0  กะ/คือ/กะ/บ่/ได้/เจ็บ/ป่วย/ด้วย/ด้วย/ตัวเอง/หน...   
39  40.0  เดี๋ยว/กะ/สิ/กิน/ไป/แต่/ว่า/กะ/สิ/ลา/ท่าน/ผู้/...   
40  41.0  นั่น/หละ/สุผู้สุคน/เดี๋ยว/กะ/น้อง/แอน/ขอ/ขอ/อน...   

                                              ภาษาไทย     เริ่ม        จบ  \
0           วัน/นี้/หนาว/มาก/ต้อง/ไป/ธุระ/โอ้ย/เอิ่ม/  00:00:05  00:00:11   
1   หิ/มะ/ไม่/ได้/ตก/หรอก/จ้า/แต่/ว่า/เป็น/น้ำ/แข็...  00:00:14  00:00:20   
2   น้ำ/แข็ง/เกาะ/น้ำ/แข็ง/เกาะ/ตั้ง/แต่/มื้อ/คื้น...  00:00:22  00:00:32   
3   ว้าย/ไม่/อยาก/ออก/ข้าง/นอก/เลย/จริง/แต่/ว่า/เอ...  00:00:33  00:00:46   
4   มือ/แข็ง/เลย/ไม่/ได้/เอา/ไม่/ได้/เอา/ถุง/มือ/ม...  00:00:48  00:00:54   
5   เอิ่ม/นี้/ก็/คือ/นั่น/แหละ/ว่า/จะ/ลา/ท่าน/ผู้/...  00:00:58  00:01:16   
6   โอ้ย/อา/กาศ/หนาว/มาก/ทุกคน/พึ่ง/นี่/เอิ่ม/เข้า...  00:01:27  00:01:38   
7   แต่/ว่า/แดด/ออก/เขา/ก็/จะ/เอ่อ/ดี/ขึ้น/นะ/คะ/ก...  00:01:39  00:01:53   
8   โอ้ย/จะ/บอก/อะ/ไร/ทุกคน/สวัสดี/โอ้ย/สวัสดี/หรื...  00:01:53  00:02:08   
9   เอิ่ม/มัน/ก็/เหลือ/หาง/มัน/นะ/คะ/ประ/มาณ/นี้/แ...  00:02:10  00:02:22   
10  ก็/ว่า/เอิ่ม/พ่อ/บ้าน/กลับ/มา/จาก/นอเวย์/แล้ว/...  00:02:23  00:02:46   
11  ที่/ไหน/ได้/นาง/ไม่/กิน/ข้าว/เย็น/ก็/คิด/ว่า/ป...  00:02:47  00:03:04   
12  ก็/ว่า/เอ่อ/เอ่อ/พ่อ/บ้าน/กิน/ถ้า/ว่า/เขา/กิน/...  00:03:06  00:03:21   
13  ก็/เลย/ได้/ข้าว/เอิ่ม/ข้าว/ไข่/เจียว/นะ/ทุกคน/...  00:03:27  00:03:31   
14  ปลา/แห้ง/ทอด/แล้ว/ก็/เป็น/น้ำ/จิ้ม/อ่า/น้ำ/จิ้...  00:03:38  00:03:54   
15  เดี๋ยว/เดี๋ยว/เดี๋ยว/เดี๋ยว/จะ/ให้/ดู/นี่/ถุ/ห...  00:04:05  00:04:25   
16  ฉัน/ก็/เลย/สั่ง/เอา/นั่น/แหละ/มา/แกะ/ถุง/เดี๋ย

In [362]:
def replacestr(col1):
    co1 = col1.replace("/", " ")
    return co1

In [363]:
replacestr(data3_df.iloc[0,1])

'มื้อ นี้ หนาว แฮง ต้อง ไป่ ธุระ โอ้ย อั่น '

In [364]:
list_1 = []
list_2 = []
for i in range(len(data1_df)):
    list_1.append(replacestr(data3_df.iloc[i,1]))
    list_2.append(replacestr(data3_df.iloc[i,2]))

In [365]:
list_1

['มื้อ นี้ หนาว แฮง ต้อง ไป่ ธุระ โอ้ย อั่น ',
 'หิ มะ บ่ ได้ ตก ดอก จ่ะ แต่ ว่า เป็น น้ำ แข็ง เลย อยาก สิ บอก ว่า ขาว ขาว จั่ง สี้ จ่ะ ',
 'น้ำ แข็ง เกาะ น้ำ แข็ง เกาะ ตั้ง แต่ มื้อ คื้น เอิ่ม ลบ สิบสี่ องศา พี่น้อง ลบ สิบ สี่ องศา',
 'วื้ย บ่ อยาก ออก ข้าง นอก เลย คั้ก แต่ ว่า อั่น มื้อ นี้ แดด ออก แต่ เช้า เดี๋ยว ขอ อนุญาต ไป เฮ็ด ธุระ ก่อน เด้อ จ่ะ นั่น หละ เดี๋ยว จั่ง ใด๋ กะ คุย กัน ใหม่',
 'มือ แข็ง เลย บ่ ได้ เอา บ่ ได้ เอา ถุง มือ มา เอา มา แต่ ว่า อั่น มัน ปิด กล้อง บ่ ได้ กะ เลย บ่ ไส่',
 'อั่น นี้ กะ คือ นั่น หละ ว่า สิ ลา ท่าน ผู้ ชม กะ อัน นี้ กะ คือ แบบ แถว แถว หม่อง อยู่ เนาะ จ่ะ เดี๋ยว กะ สิ ให้ เบิ่ง เพราะ ว่า มัน มัน เป็น น้ำ แข็ง เหมิด ละ จ่ะ แต่ ว่า หิ มะ บ่ ทัน ตก เด้อ หิ มะ สิ ตก วัน วัน พฤหัส นี้ หละ คึด ว่า นะ นั้น หละ โอ เค จ่ะ',
 'โอ้ย อา กาศ หนาว แฮง พี่น้อง เพิ่ง นี่ อั่น เข้า กับ มา บ้าน เพราะ ว่า อั่น ออก ไป ธุระ แปป นึง เนาะ จ่ะ ลบ สิบ สี่ อง ศา พี่น้อง เอ้ย สิบ สี่ อง ศา',
 'แต่ ว่า แดด ออก เพิ่น กะ สิ เอ่อ ดี ขึ้น เนาะ จ่ะ กะ หมาย ถึง ว่า อั่น อา กาศ กะ

In [366]:
list_2

['วัน นี้ หนาว มาก ต้อง ไป ธุระ โอ้ย เอิ่ม ',
 'หิ มะ ไม่ ได้ ตก หรอก จ้า แต่ ว่า เป็น น้ำ แข็ง เลย อยาก จะ บอก ว่า ขาว ขาว อย่าง นี้ จ้า ',
 'น้ำ แข็ง เกาะ น้ำ แข็ง เกาะ ตั้ง แต่ มื้อ คื้น เอิ่ม ลบ สิบสี่ องศา ทุกคน ลบ สิบ สี่ องศา',
 'ว้าย ไม่ อยาก ออก ข้าง นอก เลย จริง แต่ ว่า เอิ่ม วัน นี้ แดด ออก แต่ เช้า เดี๋ยว ขอ อนุญาต ไป ทำ ธุระ ก่อน นะ จ้ะ นั่น แหละ เดี๋ยว ยัง ไง ก็ คุย กัน ใหม่',
 'มือ แข็ง เลย ไม่ ได้ เอา ไม่ ได้ เอา ถุง มือ มา เอา มา แต่ ว่า เอิ่ม มัน ปิด กล้อง ไม่ ได้ ก็ เลย ไม่ ไส่',
 'เอิ่ม นี้ ก็ คือ นั่น แหละ ว่า จะ ลา ท่าน ผู้ ชม ก็ ตรง นี้ ก็ คือ แบบ แถว แถว ที่ อยู่ นะ คะ เดี๋ยว ก็ จะ ให้ ดู เพราะ ว่า มัน มัน เป็น น้ำ แข็ง หมด ละ จ้า แต่ ว่า หิ มะ ยัง ไม่ ตก นะ หิ มะ จะ ตก วัน วัน พฤหัส นี้ แหละ คิด ว่า นะ นั้น แหละ โอ เค จ้า',
 'โอ้ย อา กาศ หนาว มาก ทุกคน พึ่ง นี่ เอิ่ม เข้า กับ มา บ้าน เพราะ ว่า เอิ่ม ออก ไป ธุระ แปป นึง นะ คะ ลบ สิบ สี่ อง ศา ทุกคน เอ้ย สิบ สี่ อง ศา',
 'แต่ ว่า แดด ออก เขา ก็ จะ เอ่อ ดี ขึ้น นะ คะ ก็ หมาย ถึง ว่า เอิ่ม อา กาศ ก็ จะ อยู่ ประ มาณ

In [367]:
data3_df['ภาษาอีสานใหม่'] = list_1
data3_df['ภาษาไทยใหม่'] = list_2

ValueError: Length of values does not match length of index

In [368]:
data3_df

No                                          ภาษาอีสาน  \
0    1.0         มื้อ/นี้/หนาว/แฮง/ต้อง/ไป่/ธุระ/โอ้ย/อั่น/   
1    2.0  หิ/มะ/บ่/ได้/ตก/ดอก/จ่ะ/แต่/ว่า/เป็น/น้ำ/แข็ง/...   
2    3.0  น้ำ/แข็ง/เกาะ/น้ำ/แข็ง/เกาะ/ตั้ง/แต่/มื้อ/คื้น...   
3    4.0  วื้ย/บ่/อยาก/ออก/ข้าง/นอก/เลย/คั้ก/แต่/ว่า/อั่...   
4    5.0  มือ/แข็ง/เลย/บ่/ได้/เอา/บ่/ได้/เอา/ถุง/มือ/มา/...   
5    6.0  อั่น/นี้/กะ/คือ/นั่น/หละ/ว่า/สิ/ลา/ท่าน/ผู้/ชม...   
6    7.0  โอ้ย/อา/กาศ/หนาว/แฮง/พี่น้อง/เพิ่ง/นี่/อั่น/เข...   
7    8.0  แต่/ว่า/แดด/ออก/เพิ่น/กะ/สิ/เอ่อ/ดี/ขึ้น/เนาะ/...   
8    9.0  โอ้ย/สิ/บอก/อี/หยัง/พี่น้อง/สวัสดี/โอ้ย/สวัสดี...   
9   10.0  อั่น/มัน/กะ/เหลือ/หาง/มัน/เนาะ/จ่ะ/ประ/มาณ/นี้...   
10  11.0  กะ/ว่า/อั่น/พ่อ/บ้าน/กลับ/มา/จาก/นอเวย์/ละ/กะ/...   
11  12.0  ที่/ไหน/ได้/นาง/บ่/กิน/ข้าว/แลง/กะ/คึด/ว่า/ปลา...   
12  13.0  กะ/ว่า/เอ่อ/เอ่อ/พ่อ/บ้าน/กิน/คั่น/ว่า/เพิ่น/ก...   
13  14.0  กะ/เลย/ได้/ข้าว/อั่น/ข้าว/ไข่/เจียว/เด้อ/พี่น้...   
14  15.0  ปลา/แห้ง/ทอด/แล้ว/กะ/เป็น/น้ำ/จิ้ม/อ่า/น้ำ/จิ้...   
15  16.0  เดี๋ยว/เดี๋ยว/เดี๋ยว/เดี๋ยว/สิ/ไห่/เบิ่ง/นี่/โ...   
16  17.0  ข่อย/กะ/เลย/สั่ง/เอา/หั่น/น้า/มา/แกะ/ถุง/เดี๋ย...   
17  18.0  อัน/นี้/ร้าน/กี/วี/เด้อ/พี่น้อง/เป็น/ร้าน/ซุปเ...   
18  19.0  นี่/พี่น้อง/พ่อ/บ้าน/ซื้อ/ปลา/ลัฟ/มา/ให้/ปลา/ล...   
19  20.0  ซื้อ/มา/ให้/สี่/แพ็ค/พี่น้อง/เมื่อ/ก่อน/ตอน/อย...   
20  21.0  มัน/แซ/แข็ง/มา/เด้อ/จ่ะ/ซื้อ/มา/จาก/กี/วี/นอเว...   
21  22.0  แต่/ว่า/กะ/ดี/ใจ/ได้/ของ/ฝาก/หมาย/ถึง/กะ/เฮา/บ...   
22  23.0  ข่อย/กะ/เลย/ว่า/เอา/กะ/อ้าว/กะ/เลย/ว่า/จั่ง/สี...   
23  24.0  ห้า/สิบ/กว่า/โครน/เพิ่น/ว่า/จั่ง/สี้/แต่/ว่า/บ...   
24  25.0  โอ/เค้/เอา/ไป/เก็บ/เอา/ไป/เก็บ/อัน/นี่/แซ่/แซ่...   
25  26.0  รวม/กัน/ตลอด/เลย/จ่ะ/คือ/เก่า/กิน/เลย/จ่ะ/นั่น...   
26  27.0  เมื่อ/วาน/อั่น/นอเวย์/ลบ/สิบ/อง/ศา/หม่อง/ที่/ค...   
27  28.0  เศรษฐกิจ/ของ/เพิ่น/เพิ่น/สิ/มี/อ่า/ปลา/แซล/มอล...   
28  29.0  ส่ง/ออก/นอก/ประ/เทศ/เนี่ย/ซื้อ/สม/มุติ/ว่า/ซื้...   
29  30.0  บาง/คน/ถาม/มา/ใน/คอม/เมนท์/เนาะ/จ่ะ/ว่า/รส/ชาต...   
30  31.0  ไผ/สิ/มัก/กิน/ซู/ชิ/ซึ่ง/เขา/มี/ใน/ประ/กอบ/อา/...   
31  32.0  ปลา/แซล/มอล/เฮ็ด/แกง/อื้อ/ขอ/โทษ/เฮ็ด/แกง/เขีย...   
32  33.0  พ่อ/บ้าน/กลับ/มา/นอน/ตั้ง/แต่/สอง/ทุ่ง/เคิ่ง/จ...   
33  34.0  ตั้ง/แต่/ฮู้/จัก/พ่อ/บ้าน/มา/ตั้ง/แต่/มา/อยู่/...   
34  35.0  คือ/เจ็บ/ปวด/หัว/ตัว/ร้อน/จั่ง/สี้/เป็น/ไข้/บ่...   
35  36.0  เพิ่น/นั่ง/ข้าง/ข้าง/หมู่/เป็น/คน/ขับ/อั่น/ขา/...   
36  37.0  นั่น/หละ/เป็น/อุบัติเหตุ/แรก/ที่/เรา/เฮ็ด/มา/ง...   
37  38.0  ได้/พัก/สั่ม/นั้น/เอง/ตั้ง/แต่/ฮู้/จัก/กัน/มา/...   
38  39.0  กะ/คือ/กะ/บ่/ได้/เจ็บ/ป่วย/ด้วย/ด้วย/ตัวเอง/หน...   
39  40.0  เดี๋ยว/กะ/สิ/กิน/ไป/แต่/ว่า/กะ/สิ/ลา/ท่าน/ผู้/...   
40  41.0  นั่น/หละ/สุผู้สุคน/เดี๋ยว/กะ/น้อง/แอน/ขอ/ขอ/อน...   

                                              ภาษาไทย     เริ่ม        จบ  \
0           วัน/นี้/หนาว/มาก/ต้อง/ไป/ธุระ/โอ้ย/เอิ่ม/  00:00:05  00:00:11   
1   หิ/มะ/ไม่/ได้/ตก/หรอก/จ้า/แต่/ว่า/เป็น/น้ำ/แข็...  00:00:14  00:00:20   
2   น้ำ/แข็ง/เกาะ/น้ำ/แข็ง/เกาะ/ตั้ง/แต่/มื้อ/คื้น...  00:00:22  00:00:32   
3   ว้าย/ไม่/อยาก/ออก/ข้าง/นอก/เลย/จริง/แต่/ว่า/เอ...  00:00:33  00:00:46   
4   มือ/แข็ง/เลย/ไม่/ได้/เอา/ไม่/ได้/เอา/ถุง/มือ/ม...  00:00:48  00:00:54   
5   เอิ่ม/นี้/ก็/คือ/นั่น/แหละ/ว่า/จะ/ลา/ท่าน/ผู้/...  00:00:58  00:01:16   
6   โอ้ย/อา/กาศ/หนาว/มาก/ทุกคน/พึ่ง/นี่/เอิ่ม/เข้า...  00:01:27  00:01:38   
7   แต่/ว่า/แดด/ออก/เขา/ก็/จะ/เอ่อ/ดี/ขึ้น/นะ/คะ/ก...  00:01:39  00:01:53   
8   โอ้ย/จะ/บอก/อะ/ไร/ทุกคน/สวัสดี/โอ้ย/สวัสดี/หรื...  00:01:53  00:02:08   
9   เอิ่ม/มัน/ก็/เหลือ/หาง/มัน/นะ/คะ/ประ/มาณ/นี้/แ...  00:02:10  00:02:22   
10  ก็/ว่า/เอิ่ม/พ่อ/บ้าน/กลับ/มา/จาก/นอเวย์/แล้ว/...  00:02:23  00:02:46   
11  ที่/ไหน/ได้/นาง/ไม่/กิน/ข้าว/เย็น/ก็/คิด/ว่า/ป...  00:02:47  00:03:04   
12  ก็/ว่า/เอ่อ/เอ่อ/พ่อ/บ้าน/กิน/ถ้า/ว่า/เขา/กิน/...  00:03:06  00:03:21   
13  ก็/เลย/ได้/ข้าว/เอิ่ม/ข้าว/ไข่/เจียว/นะ/ทุกคน/...  00:03:27  00:03:31   
14  ปลา/แห้ง/ทอด/แล้ว/ก็/เป็น/น้ำ/จิ้ม/อ่า/น้ำ/จิ้...  00:03:38  00:03:54   
15  เดี๋ยว/เดี๋ยว/เดี๋ยว/เดี๋ยว/จะ/ให้/ดู/นี่/ถุ/ห...  00:04:05  00:04:25   
16  ฉัน/ก็/เลย/สั่ง/เอา/นั่น/แหละ/มา/แกะ/ถุง/เดี๋ย